doc/wiki.txt

```diff
@@ -238,13 +238,31 @@ CONFIGURATION                                                     *wiki-config*
         \ },
         \}
 
-*g:wiki_mappings_use_defaults*
+
+*g:wiki_mappings_use_all_defaults*
   Whether or not to use default mappings (see |wiki-mappings-default|). If
+  Whether or not to use all default mappings (see |wiki-mappings-default|). If
+  this is set to 0, then only custom mappings will be applied. Custom mappings
   this is set to 0, then only custom mappings will be applied. Custom mappings
   may either be specified explicitly or through |g:wiki_mappings_global| and
+  may either be specified explicitly or through |g:wiki_mappings_global| and
   |g:wiki_mappings_local|.
+  |g:wiki_mappings_local|.
+
 
   Default: 1
+  Default: 1
```

Comment:
The above seems unfinished. A lot of duplicate lines...

Reply:
Whoops - I have no idea how these lines got duplicated... Thanks for the catch.

Reply:
Fixed in a06e342